### DISCUSSION TITLE
Golang Guild Session: Concurrency Primitives | Sakharam Shinde

### DISCUSSION THREAD

**Discussion by giscus** [llorllale/llorllale.github.io#58] [Announcements]
# Golang Guild Session: Concurrency Primitives | Sakharam Shinde

Slide deck for my presentation on Golang Concurrency Primitives for the Golang Guild Session @ VerticalScope.

http://127.0.0.1:5501/posts/golang-concurrency-primitives/index.html

**Comment by SakharamS** [comment]
Test Comment
